airbyte-server/src/main/java/io/airbyte/server/handlers/SchedulerHandler.java

```diff
@@ -75,6 +85,7 @@
 public class SchedulerHandler {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(SchedulerHandler.class);
+  private static final HashFunction hashFunction = Hashing.md5();
```

Comment:
nit: change the name of this constant to be `HASH_FUNCTION` (snake case is our convention for constant names)